internal/config/config.go

```diff
@@ -502,6 +519,14 @@ func (c *Config) Validate() error {
 		return err
 	}
 
+	versionCompatibilityValidator := validateVersionCompatibility
+	if force {
+		versionCompatibilityValidator = returnsTrue
+	}
+	if err := validate.RegisterValidation("version_compatibility", versionCompatibilityValidator); err != nil {
+		return err
+	}
```

Comment:
```suggestion
	if force {
		if err := validate.RegisterValidation("version_compatibility", validateVersionCompatibility); err != nil {
			return err
		}
	}
```
I think this is a bit simpler.
We can gate all "optional" validation behind the `force` flag, and just not register them if it is unset.
Needs to be tested if this actually works, the validator might throw an error for undefined validations if we specify one but never register it, not sure.